gen3.datacommons.io/portal/gitops.json

```diff
@@ -225,47 +280,67 @@
       },
       "fieldsToShow": [
         {
-          "groupName": "OADC studies",
+          "groupName": "Gen3 Studies",
           "includeName": false,
           "fields": [
             {
-              "name": "Accession Number",
-              "field": "accession_number"
+              "name": "Study Name",
+              "field": "full_name",
+              "contentType": "string",
+              "includeIfNotAvailable": false
             },
             {
               "name": "Study Title",
-              "field": "study_title"
```

Comment:
@cgmeyer - like here. Looks like there was not a comma previously, and you added one?

Reply:
Oh wait -- is the reason there wasn't one because it was the last item in the list of fields in the bracket before, and you also added more fields, so it needed a comma at the end?